int21.c

```diff
@@ -138,6 +139,22 @@ void int21(uc_engine *uc)
             //printf("\n>>> 0x%x: interrupt: %x, AH = %02x\n", r_ip, 0x21, r_ah);
             break;
 
+        case 0x00: //terminate process
+            {
+                uc_emu_stop(uc);
+            }
+
+        case 0x02: // character output
+            {
+                uint8_t r_dl;
+
+                uc_reg_read(uc, UC_X86_REG_DL, &r_dl);
+
+                printf("%c\n", r_dl);
```

Comment:
You are right, i haven't considered that this function can be used to print a string like putc in c.